aeromaps/models/base.py

```diff
@@ -33,6 +51,18 @@ def _initialize_df(self):
         self.xarray_lca: xr.DataArray = xr.DataArray()
         self.years = np.linspace(self.historic_start_year, self.end_year, len(self.df.index))
 
+    def _store_outputs(self, output_data):
+        """
+        Store vector outputs in self.df and float outputs in self.float_outputs
+        """
```

Comment:
Could this be used by all models or is it useful only for some models? For example is it valid only for custom type models?